protocol/implementations/js/src/validator.ts

```diff
@@ -0,0 +1,34 @@
+import Ajv from 'ajv'
+import addFormats from 'ajv-formats'
+
+import tbdexMessage from '../../json-schemas/tbdex-message.schema.json' assert { type: 'json' }
```

Comment:
@mistermoe this is causing the node tests to fail on build with:
`Import assertions are only supported when the '--module' option is set to 'esnext' or 'nodenext'.`, which is confusing because the root `tsconfig.json` does set module to esnext? 

Also doesn't happen ✨ on my machine ✨

Reply:
Looking at tsconfig again, does this include block:
```
  "include": [
    "./src",
  ],
  ```
potentially mean that it can't find `/protocol/implementations/json-schema` ? (although the error doesn't indicate that)

Reply:
ah yup! it's being caused by `module` being set to `CommonJS` in `tsconfig.cjs.json` which was a flub on my part. just fixed and pushed.